docs/introduction.md

```diff
@@ -4,145 +4,62 @@
+
+On the other hand, abstracting Kubernetes to serve developers' needs is a highly opinionated process, and the resultant abstractions would only make sense had the decision makers been the platform builders. Unfortunately, the platform builders today face the following dilemma:
+
+*There is no tool or framework for them to easily extend the abstractions if any*. 
+
+Thus, many platforms today introduce restricted abstractions, implementations and add-on mechanisms despite the extensibility of Kubernetes. This makes extending such platforms per developers' requirements or to wider scenarios almost impossible, not to mention taking full advantage of the extremely rich Kubernetes ecosystems. . 
+
+In the end, developers complain such a platform is too rigid and slows in response to feature requests or improvements. The platform engineers do want to help but the engineering cost is daunting: any simple API change in the platform could easily become a marathon negotiation around the opinionated abstraction design.
 
 ## What is KubeVela?
 
-KubeVela is a framework to help platform builders to easily expose and extend
-Kubernetes capabilities for application management. It is built on top of
-Kubernetes and relieves the pains of both the platform builders and the application
-developers by doing the following:
+For developers, KubeVela itself is an easy-to-use tool that enables them to describe and ship their applications to Kubernetes with minimal effort. Instead of managing a handful Kubernetes YAML files, a simple docker-compose style [Appfile](./docs/developers/devex/appfile.md) is what all they need, following an application-centric workflow that can be easily integrated with any CI/CD pipeline.
 
-- KubeVela enforces a single **application** concept and **ALL** the exposed
-  Kubernetes capabilities serve for the applications only.
-  It adopts the [Open Application Model](https://github.com/oam-dev/spec) (OAM)
-  for its application definition. The conventional Pod and container concepts
-  are completely eliminated.
+For platform builders, KubeVela serve as a framework that empowered platform teams create developer facing yet highly extensible platforms at ease. In detail, KubeVela relieves the pains of building such platforms by doing the following:
```

Comment:
- that empowered platform teams create -> that empowers platform teams to create
- In detail -> in details